src/reducers/mainReducer.js

```diff
@@ -0,0 +1,42 @@
+import * as types from '../actions/actionTypes';
+
+const initialState = {
+    movies: [],
+    titleActive: true,
+    sortByReleaseDate: true,
+    searchText: ''
+};
+export default function mainReducer(state = initialState, action) {

```

Comment:
I believe one reducer do the job for you as your domain is basically the same.

Reply:
Done, there is now a single reducer file and single action creator file.